docs/topics/http/views.txt

```diff
@@ -60,12 +60,12 @@ date and time. To display this view at a particular URL, you'll need to create a
 Returning errors
 ================
 
-Returning HTTP error codes in Django is easy. There are subclasses of
+Django provides help for returning HTTP error codes: There are subclasses of
```

Comment:
Full stop (period) instead of colon would make the sentences shorter, easier to read, and easier to translate.